modules/nw-sriov-dpdk-example-intel.adoc

```diff
@@ -66,17 +66,20 @@ metadata:
   namespace: openshift-sriov-network-operator
 spec:
   networkNamespace: <target_namespace>
-  ipam: "{}" <1>
+  ipam: |-
+# ... <1>
   vlan: <vlan>
   resourceName: intelnics
 ----
-<1> Specify an empty object `"{}"` for the ipam CNI plug-in. DPDK works in userspace mode and does not require an IP address.
+<1> Specify a configuration object for the ipam CNI plug-in as a YAML block scalar. The plug-in manages IP address assignment for the attachment definition.
```

Comment:
Do we need an example here, or will our users understand how to configure this?

Reply:
note 1 contains a reference to a code example for a configuration already